Mimir/Repositories/TableSheetsRepository.cs

```diff
@@ -13,20 +18,32 @@ public class TableSheetsRepository(MongoDBCollectionService mongoDbCollectionSer
 {
     protected override string GetCollectionName() => "table_sheet";
 
-    public async Task<(int, int)> GetLatestArenaSeason(string network, long blockIndex)
```

Comment:
It is just a question: Why does this method become a synchronous method?

Reply:
I changed it to a synchronous method by changing the return type without thinking about it. It's better to leave it asynchronous.😅
I'll try to convert other `XyzRepository`s to asynchronous in the future.